tests/networkpolicy_test.go

```diff
@@ -176,19 +238,26 @@ func skipNetworkPolicyRunningOnKindInfra() {
 	}
 }
 
-func createVMICirros(virtClient kubecli.KubevirtClient, namespace string, labels map[string]string) *v1.VirtualMachineInstance {
+func createVMICirros(virtClient kubecli.KubevirtClient, namespace string, labels map[string]string, opts ...libvmi.Option) *v1.VirtualMachineInstance {
 	var err error
 	vmi := tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskCirros), "#!/bin/bash\necho 'hello'\n")
 	vmi.Namespace = namespace
 	vmi.Labels = labels
+
+	// Clean up interfaces since we configure them with `libvmi.Option`
+	vmi.Spec.Domain.Devices.Interfaces = []v1.Interface{}
```

Comment:
I wonder why you're only replacing the interface when creating VMIs via this function (why not override the network as well ?).

IIUC, this would only work if the name of the interface you're creating is 'default'. Can't say that I like that implicit requirement. 

Maybe this is not a real concern, since it looks like this assumption - both network and ifaces are called default - plagues our code base everywhere; I might be taking this a bit too far, what you're doing is not worse that what we currently have.